.github/workflows/e2e-test.yaml

```diff
@@ -0,0 +1,20 @@
+name: Run CCNP E2E test
+on:
+   schedule:
+    - cron: '25 7 * * *'
+   workflow_dispatch:
+jobs:
+  E2E_test:
+    runs-on: tdvm-ut
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.1.1
+      - name: Run Test
+        run: |
+           cd ./test/e2e-test/
+           ./ci-test.sh
+      - name: Clean up the environment
+        if: always()
+        run: |
+           cd ./test/e2e-test/
```

Comment:
I remember @LeiZhou-97 once submitted a comment about the scripts' directory location, is it "hack" more proper or something like a BKM?
current way is acceptable for me, but still some minor suggestions:
1. "test/e2e-test" =>"tests/e2e"
2. Since this is e2e test, not sure if there will be other test cases in CI? So better change "ci-test.sh" => "e2e-test.sh", this will give most flexibility for future test case extension.
3. Still the original concern: if we need to extend test cases later, it is better to have a script similar as "ci-setup.sh" for common CI test environment initialization, then "ci-xxx-test.sh" and finally "ci-cleanup.sh", if we do not need to care about more test cases, just keep all cases together in "ci-tests.sh", it is acceptable for me, it depends on community and team decision. 
4. combine the two lines to single line in "run" step, like: ./tests/e2e/ci-test.sh,        ./tests/e2e/ci-clean.sh

Reply:
@jiere 
I mean if we have some scripts that are related to our CI/CD work can put under /hack directory, which is a cloud native popular pattern for external engineers build, dev and test our CCNP.

For just test cases, I think it's better to put under /test.

Reply:
Fixed.
The next PR will change the directory of the scripts, including some other scripts of CCNP.

Reply:
Exactly, we had an offline discussion, and we found k8s style directory layout introduction like here: https://zhuanlan.zhihu.com/p/645269730